build/math-cert-tests/math-code-test-a.py

```diff
@@ -255,7 +255,7 @@ def step22():
 def step23(code):
     import re
     print(" ")
-	code_count = 0
+        code_count = 0
```

Comment:
My editor is showing that the indentation here is unexpected, and doesn't match the `code_count` declarations in the other functions. This seems to be the fix:

```suggestion
    code_count = 0
```